.github/workflows/project-automation.yml

```
@@ -0,0 +1,103 @@
+
+            await github.rest.issues.addLabels({
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              issue_number: context.payload.issue.number,
+              labels: [sizeLabel]
+            });
+
+  priority-notification:
+    name: Notify on High Priority
+    runs-on: ubuntu-latest
+    if: contains(github.event.issue.labels.*.name, 'priority: critical') || contains(github.event.issue.labels.*.name, 'priority: high')
```

actionlint (1.7.9)

could not parse as YAML: mapping values are not allowed in this context

(syntax-check)